terraform/gateway-host.tf

```diff
@@ -1,3 +1,9 @@
+variable "gateway_host_name" {
```

Comment:
As other instance names are declared in environment vars, shouldn't it be the same here?